travis-ci/vmtest/helpers.sh

```diff
@@ -2,11 +2,23 @@
 # $2 - fold identifier, no spaces
 # $3 - fold section description
 travis_fold() {
-	local YELLOW='\033[1;33m'
-	local NOCOLOR='\033[0m'
-	echo travis_fold:$1:$2
-	if [ ! -z "${3:-}" ]; then
-		echo -e "${YELLOW}$3${NOCOLOR}"
-	fi
-	echo
+  local YELLOW='\033[1;33m'
+  local NOCOLOR='\033[0m'
+  if [ -z ${GITHUB_WORKFLOW+x} ];
```

Comment:
did you mean to use
```
${parameter:+word}

    If parameter is null or unset, nothing is substituted, otherwise the expansion of word is substituted.
```
?

Reply:
Doest really matter -
with :+ it will do check for var is unset or empty
with + it will check only if var is unset
```
[siudin@devvm2769.vll0 ~/local/gh/vmtest] qwe=""
[siudin@devvm2769.vll0 ~/local/gh/vmtest] if [ -z ${qwe+x} ]; then echo 1 ; fi
[siudin@devvm2769.vll0 ~/local/gh/vmtest] if [ -z ${qwe:+x} ]; then echo 1 ; fi
1
[siudin@devvm2769.vll0 ~/local/gh/vmtest] unset qwe
[siudin@devvm2769.vll0 ~/local/gh/vmtest] if [ -z ${qwe:+x} ]; then echo 1 ; fi
1
[siudin@devvm2769.vll0 ~/local/gh/vmtest] if [ -z ${qwe+x} ]; then echo 1 ; fi
1
```

Reply:
oh, cool, I couldn't find documentation for +x case, so I was worried that it's a typo. Never mind :)